gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -162,6 +164,9 @@
   // https://cloud.google.com/storage/docs/json_api/v1/objects#resource-representations
   private static final String OBJECT_FIELDS = "bucket,name,size,updated";
 
+  // To track the http requests from GoogleCloudStorage
+  static GcsioTrackingHttpRequestInitializer gcsRequestsTracker;
```

Comment:
Do we need an instance in the GCS connector layer, why not account from gcsio ?

Reply:
Updated